src/Helpers/Form/Utils.php

```diff
@@ -46,9 +46,9 @@ function isProcessingForm() {
  * @return bool
  */
 function isProcessingGiveActionOnAjax() {
-	return isset( $_REQUEST['action'] ) &&
-		   wp_doing_ajax() &&
-		   0 === strpos( $_REQUEST['action'], 'give_' );
+	$action            = isset( $_REQUEST['action'] ) ? give_clean( $_REQUEST['action'] ) : '';
+	$whiteListedAction = [ 'get_receipt' ];
+	return $action && wp_doing_ajax() && ( 0 === strpos( $action, 'give_' ) || in_array( $action, $whiteListedAction, true ) );
```

Comment:
I'm confused, is this related to limiting the decimal places?

Reply:
@JasonTheAdams Yes, this is required. Basically form template was not loading when fetching receipt by ajax that's why hooks were not working. It will allow the loading form template on ajax. `get_reciept` is a special case in Give core where `action` parameter  (required for WP Ajax) does not have `give_` prefix.